lib/order.rb

```diff
@@ -0,0 +1,53 @@
+#understands an order
+
+class Order
+
+  def initialize
+    @order_content = Hash.new(0)
+    @checkout_value = nil
+  end
+
+  #add dish with quantity and sum to the order
```

Comment:
too many comments - it becomes hard to manage the comments as you are coding - methods change, and their purpose changes, it's easy to leave misleading comments. Let your tests be your documentation.